src/Kernel32.Desktop/Kernel32+GRPICONDIRENTRY.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="GRPICONDIRENTRY"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// Represents an icon as stored in a resource
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, Pack=1)]
```

Comment:
nit: Would `Pack=1` reformat to `Pack = 1` if you execute a Format Document command? If so, can you do that now so that folks who change the file in the future won't see a diff with extra changes in it?